<?xml version="1.0" encoding="UTF-8"?>
<testsuite
  name="tbot_start.flake8"
  tests="4"
  errors="0"
  failures="4"
  time="0.059"
>
  <testcase
    name="E302 (./launch/bringup.launch.py:7:1)"
    classname="tbot_start.flake8"
  >
      <failure message="expected 2 blank lines, found 1:&#10;def generate_launch_description():&#10;"/>
  </testcase>
  <testcase
    name="E231 (./launch/bringup.launch.py:9:45)"
    classname="tbot_start.flake8"
  >
      <failure message="missing whitespace after ',':&#10;    kobuki_urdf = os.path.join(tbot_sim_path,'urdf', 'kobuki_standalone.urdf')&#10;"/>
  </testcase>
  <testcase
    name="E128 (./launch/bringup.launch.py:43:17)"
    classname="tbot_start.flake8"
  >
      <failure message="continuation line under-indented for visual indent:&#10;                'robot_description': urdf}]),&#10;"/>
  </testcase>
  <testcase
    name="E302 (./launch/move_launch.py:4:1)"
    classname="tbot_start.flake8"
  >
      <failure message="expected 2 blank lines, found 1:&#10;def generate_launch_description():&#10;"/>
  </testcase>
  <system-out>Checked files:
* ./launch/bringup.launch.py
* ./launch/move_launch.py</system-out>
</testsuite>
